test/utils/form.ts

```diff
@@ -0,0 +1,296 @@
+import { Locator } from '@playwright/test';
+import { DashboardPage, expect, Panel, PanelEditPage } from '@grafana/plugin-e2e';
+import { getLocatorSelectors, LocatorSelectors } from './selectors';
+import { TEST_IDS } from '../../src/constants/tests';
+
+const getElementsSelector = getLocatorSelectors(TEST_IDS.formElements);
+
+/**
+ * Disabled Element Helper
+ */
+class DisabledElementHelper {
+  private readonly locator: Locator;
+
+  constructor(parentLocator: Locator) {
+    this.locator = parentLocator.getByTestId(TEST_IDS.formElements.fieldDisabled);
+  }
+
+  private getMsg(message: string): string {
+    return `DisabledElement: ${message}`;
+  }
+
+  public get() {
+    return this.locator;
+  }
+
+  public async checkValue(text: string) {
+    return expect(this.get(), this.getMsg('Check Text')).toHaveValue(text);
+  }
+}
+
+/**
+ * Number Element Helper
+ */
+class NumberElementHelper {
+  private readonly locator: Locator;
+
+  constructor(parentLocator: Locator) {
+    this.locator = parentLocator.getByTestId(TEST_IDS.formElements.fieldNumber);
+  }
+
+  private getMsg(message: string): string {
+    return `NumberElement: ${message}`;
+  }
+
+  public get() {
+    return this.locator;
+  }
+
+  public async checkValue(text: string) {
+    return expect(this.get(), this.getMsg('Check value')).toHaveValue(text);
+  }
+
+  public async setValue(value: string) {
+    await this.get().fill(value);
+    return this.get().blur();
+  }
+
+  public async isDisabled() {
+    return expect(this.get(), this.getMsg('Number element is Disabled')).toBeDisabled();
+  }
+
+  public async isNotDisabled() {
+    return expect(this.get(), this.getMsg('Number element is not Disabled')).not.toBeDisabled();
+  }
+}
+
+/**
+ * Select Element Helper
+ */
+class SelectElementHelper {
+  private readonly locator: Locator;
+  private readonly page: DashboardPage;
+
+  constructor(parentLocator: Locator, page: DashboardPage) {
+    this.locator = parentLocator.getByTestId(TEST_IDS.formElements.fieldSelect);
+    this.page = page;
+  }
+
+  public get() {
+    return this.locator;
+  }
+
+  public async setValue(fieldKey) {
+    await this.get().click();
+    return this.page.getByGrafanaSelector(this.page.ctx.selectors.components.Select.option).getByText(fieldKey).click();
+  }
+}
+
+/**
+ * Elements Helper
+ */
+class ElementsHelper {
+  public selectors: LocatorSelectors<typeof TEST_IDS.formElements>;
+
+  constructor(public readonly locator: Locator) {
+    this.selectors = this.getSelectors(locator);
+  }
+
+  private getMsg(msg: string): string {
+    return `Elements: ${msg}`;
+  }
+
+  private getSelectors(locator: Locator) {
+    return getElementsSelector(locator);
+  }
+
+  public async checkPresence() {
+    return expect(this.selectors.root(), this.getMsg('Elements Container Presence')).toBeVisible();
+  }
+
+  public async checkElementPresence(elementId: string, elementType: string) {
```

Comment:
please use enum FormElementType for elementType instead of string - https://github.com/VolkovLabs/business-forms/blob/274dcc72be1c131d1b39113754e3a4b2e67f1a9e/src/constants/form-element.ts#L10